src/Avalonia.Input/IKeyboardDevice.cs

```diff
@@ -27,13 +27,8 @@ public enum KeyStates
         Toggled = 2,
     }
 
-    public interface IKeyboardDevice : IInputDevice, INotifyPropertyChanged
+    public interface IKeyboardDevice : IInputDevice
```

Comment:
Making this implement `INotifyPropertyChanged` to notify changes to `FocusedElement` feels inconsistent with the addition of the `FocusManager.FocusedElementChanged` event elsewhere in this PR.

Reply:
Ehm, I've *removed* that interface

Reply:
Oh crap, so you have. Sorry, need to learn to computer.